### PR TITLE
Fix relative path handling for autoload and schema draft file

### PR DESCRIPTION
**Pull request by liquorvicar**

### CODE REVIEW

File: bin/php-json-schema-generate [hunk 1]
```diff
@@ -1,10 +1,17 @@
 #!/usr/bin/env php
 <?php
 
-if (is_file(__DIR__.'/../autoload.php') === true) {
-    include_once __DIR__.'/../autoload.php';
-} elseif (is_file(__DIR__.'/../vendor/autoload.php') === true) {
-    include_once __DIR__.'/../vendor/autoload.php';
+$vendorDir = '';
+
+foreach (array(__DIR__ . '/../../../', __DIR__ . '/../vendor/', __DIR__ . '/vendor/') as $dir) {
```
Comment (MarcusNoble) [inline bin/php-json-schema-generate]: Is this right? Looks like it going up a lot of levels :/

**Comment by liquorvicar** [comment]
@benb1983 Yes, that was irking me. Have removed some of the duplication...

File: bin/php-json-schema-generate (hunk 1 above)
Comment (liquorvicar) [reply bin/php-json-schema-generate]: Yes, that's right. When it's installed as a composer dependency in another project it lives at `vendor/elsevier-io/json-schema-php-generator/bin/php-json-schema-generate` and it needs the autoload script at `vendor/autoload.php`

**Comment by benb1983** [comment]
Nice, much better!

**Pull request merged**